spec/graphql/mutations/invites/create_spec.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Mutations::Invites::Create, type: :graphql do
+  let(:membership) { create(:membership) }
+  let(:organization) { membership.organization }
+
+  let(:mutation) do
+    <<~GQL
+      mutation($input: CreateInviteInput!) {
+        createInvite(input: $input) {
+          id
+          token
+          email
+        }
+      }
+    GQL
+  end
+
+  it 'creates an invite for a new user' do
+    email = Faker::Internet.email
+
+    result = execute_graphql(
+      current_user: membership.user,
```

Comment:
you can use a `let(:payload)` to declare this too, it's better to have small `it` blocks

Reply:
Not sure how to make it shorter. Didn't found the way to `.merge` the two hash.